Example/Pods/Local Podspecs/WillowTreeReachability.podspec.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "WillowTreeReachability",
+  "version": "2.0.1",
+  "summary": "Simple Swift class for monitoring network reachability.",
+  "description": "Simple Swift class for monitoring network reachability. This class uses a subscription model to notify listeners of network status changes. Multiple listeners may be added to a single Reachability instance.",
+  "homepage": "https://github.com/willowtreeapps/WillowTreeReachability",
+  "license": {
+    "type": "MIT",
+    "file": "LICENSE"
+  },
+  "authors": "WillowTree, Inc.",
+  "platforms": {
+    "ios": "8.0"
+  },
+  "source": {
+    "git": "https://github.com/willowtreeapps/WillowTreeReachability.git",
+    "tag": "2.0.0"
```

Comment:
Yeah - it's an auto generated Pod file but I saw that after the fact, good catch.